README.md

```diff
@@ -1,2 +1,128 @@
-# kpsr-core-benchmarks
-Klepsydra core benchmarks
+<p align="right">
```

Comment:
this readme is a copy of kpsr-core. Make it oriented to the actual repo.

Reply:
Done.